### DISCUSSION TITLE
Essential and Pinned tabs sync

### DISCUSSION THREAD

**Discussion by filiptrplan** [zen-browser/desktop#3088] [Ideas]
It would be great if the "permanent" (essential and pinned) tabs also synced across computers (and windows!) I saw that the pinned tabs are planned, but wanted to add the window syncing too.

**Comment by theDanielJLewis** [comment]
Ditto.

This is a dealbreaker for me because I don't want to have to set up all my pinned and essential tabs twice (first moving from Arc, second replicating to other PC).

**Comment by frozdbyte** [comment]
Totally agree here! Setting up pinned tabs again and again is really annoying.

**Comment by nonvenomous** [comment]
I had to setup a new PC and I have one workspace with lots of non changing pinned tabs and my workaround for was

1. select relevant (pinned) tabs 
2. right click "Bookmark Tabs"
3. create some folder e.g. "Research Workspace"
4. -- change to different zen instance
5. switch to workspace
6. middle click folder to open all tabs
7. select all tabs
8. right click "Pin Tabs"

I assume this workflow would be quite easy to automate with an extension. But this seems more like it should be a native sync feature.

**Comment by Hikiru** [comment]
bump, old issue but still a big one

**Comment by jd-santos** [comment]
Has this been implemented or planned? I'd love to give up on the carcass of Arc, but I hop between computers a lot and need my browser setup to go with me.
